packages/oidc/src/refresh/refreshGrant.spec.ts

```diff
@@ -63,16 +63,20 @@ describe("refreshGrant", () => {
     );
   });
 
-  it("does not use basic auth if no client secret is available", async () => {
+  it("include the client id in the body if no client secret is available", async () => {
```

Comment:
Make sure you also test when client_id is not an IRI, e.g., someone configured using client credentials but forgot to set client_secret's value; in that case I don't think client_id should be sent in the body.

Reply:
d218289054cd254c39a7c8c7d67cc568d2b5ecd1